### DISCUSSION TITLE
Не работает динамическая локализация

### DISCUSSION THREAD

**Discussion by DendiRob** [yandex-maps-unofficial/vue-yandex-maps#379] [Q&A (2.x)] [answered]
Пытаюсь настроить локализацию на nuxt 3, использую VueYandexMaps.settings.value(https://yandex-maps-unofficial.github.io/vue-yandex-maps/guide/configuration.html#доступ-к-конфигурации), но карта не обновляется с актуальными данными

**Comment by daniluk4000** [comment, marked as answer]
На данный момент реактивная локализация не была реализована. Можете открыть feature request с этим

**Reply by daniluk4000** [reply]
Поправка. https://github.com/yandex-maps-unofficial/vue-yandex-maps/issues/206

**Reply by DendiRob** [reply]
Понял, спасибо
